tests/testament/categories.nim

```diff
@@ -17,11 +17,12 @@ const
   rodfilesDir = "tests/rodfiles"
 
 proc delNimCache(filename, options: string) =
-  let dir = nimcacheDir(filename, options)
-  try:
-    removeDir(dir)
-  except OSError:
-    echo "[Warning] could not delete: ", dir
+  for target in [targetC, targetCpp, targetObjC, targetJS]:
```

Comment:
Is there a better way to loop over an enum? Ideally I wouldn't need to list all targets here explicitly but I don't know a better way.

Reply:
low(MyEnum)..high(MyEnum)

Reply:
Oh, I assumed that would only work on Ordinals and I don't think TTarget is. But it seems to work and does the right thing, so I changed the commit to do that.

Reply:
Just reread the enums manual section again and I get it now. `x = "foo"` doesn't make an enum not an ordinal because it doesn't affect the numeric value, only the result of `$`. I had (mistakenly) assumed it was making the base type for the enum a string rather than an int.

Reply:
Just `for target in TTarget` works as well.